lighthouse-core/config/default.js

```diff
@@ -13,8 +13,8 @@ module.exports = {
     passName: 'defaultPass',
     recordTrace: true,
     pauseAfterLoadMs: 5250,
-    networkQuietThresholdMs: 5000,
-    pauseAfterNetworkQuietMs: 2500,
+    networkQuietThresholdMs: 5250,
```

Comment:
Is the idea here to realign `networkQuietThresholdMs` and the new `cpuQuietThresholdMs` with the `pauseAfterLoadMs` delay instead of the lower pause thresholds previously being used?